iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -98,6 +105,9 @@ public override async Task OpenAsync(CancellationToken cancellationToken)
             {
                 using var ctb = new CancellationTokenBundle(_operationTimeout, cancellationToken);
                 await _amqpUnit.OpenAsync(ctb.Token).ConfigureAwait(false);
+
+                // Create a timer to remove any expired messages. The timer would invoke callback after every hour.
+                _twinTimeoutTimer = new Timer(CheckTimeout, null, s_twinResponseTimeout, s_twinResponseTimeout);
```

Comment:
We're leaking here. Can you initialize an instance in ctor, and then use the `Change` method to get it going here?

Reply:
Good catch. Thanks.
Added the change.